src/realizations/catchment/Bmi_Py_Adapter.cpp

```diff
@@ -239,8 +239,9 @@ py::array Bmi_Py_Adapter::parse_other_var_val_for_setter(const geojson::JSONProp
         // TODO: think about handling list explicitly, and handling object type; for now, document restrictions
         //else if (other_value_json.get_type() == geojson::PropertyType::String) {
     else {
-        return pybind11::array(py::dtype::of<string>(), {1, 1}, {other_value_json.as_string()});
+        //return pybind11::array(py::dtype::of<string>(), {1, 1}, {other_value_json.as_string()});
```

Comment:
@robertbartel I get build errors with these uncommented. I can make updates if you have any suggestions. I put the temp var 'a' below in its place.

Reply:
/realizations/catchment/Bmi_Py_Adapter.cpp:242:54:   required from here

extern/pybind11/include/pybind11/numpy.h:1160:37: error: static assertion failed: Attempt to use a non-POD or unimplemented POD type as a numpy dtype
 1160 |     static_assert(is_pod_struct<T>::value, "Attempt to use a non-POD or unimplemented POD type as a numpy dtype");

Reply:
@jdmattern-noaa, for now, I'll just accept the changes and do some testing/adjusting in my branch before doing a PR to `upstream`.